### DISCUSSION TITLE
multiple wallet address management

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#6014] [Support]
After clicking the New Address button, the address would change, and the first address would disappear from the GUI, and never able to get back. Yet in CLI when using "chia keys show" command, only the first wallet address is listed. This is very confusing.

(1) Need to be able to pick the first address back in the GUI.
(2) Need to be able to list, and delete any unwanted addresses.

Thanks.

**Comment by BrandtH22** [comment]
@pluto2021 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - as you have identified the first wallet address can be captured from the cli command and unfortunately the nature of key pairs and derived addresses would make it unwise to "delete unwanted addresses" as it could be the case that funds are sent to those addresses and a deletion would remove visibility of those funds.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
